source/features/repo-header-info.tsx

```diff
@@ -0,0 +1,63 @@
+import * as pageDetect from 'github-url-detection';
+import {RepoForkedIcon, StarIcon} from '@primer/octicons-react';
+import React from 'dom-chef';
+import {CachedFunction} from 'webext-storage-cache';
+
+import observe from '../helpers/selector-observer.js';
+import features from '../feature-manager.js';
+import api from '../github-helpers/api.js';
+import GetRepositoryInfo from './repo-header-info.gql';
+import {cacheByRepo} from '../github-helpers/index.js';
+import abbreviateNumber from '../helpers/abbreviate-number.js';
+
+type RepositoryInfo = {
+	isFork: boolean;
+	stargazerCount: number;
+};
+
+const repositoryInfo = new CachedFunction('stargazer-count', {
+	async updater(): Promise<RepositoryInfo> {
+		const {repository: {isFork, stargazerCount}} = await api.v4(GetRepositoryInfo);
+
+		return {isFork, stargazerCount};
+	},
+	maxAge: {days: 1},
+	staleWhileRevalidate: {days: 3},
+	cacheKey: cacheByRepo,
+});
+
+async function add(repoLink: HTMLAnchorElement): Promise<void> {
+	const {isFork, stargazerCount} = await repositoryInfo.get();
+
+	if (isFork) {
+		repoLink.append(
+			<RepoForkedIcon className="v-align-text-bottom ml-1" width={12} height={12}/>,
```

Comment:
Is the alignment necessary? In my test the lock did not have other classes.

I'm ok with this class if it means we can continue using this simple .append() and .after() call